CacheTagsBundle/Listener/ResponseListener.php

```diff
@@ -11,7 +11,7 @@
 use Symfony\Component\HttpKernel\Event\FilterResponseEvent;
 use Symfony\Component\HttpKernel\HttpKernelInterface;
 
-class Response
+class ResponseListener
```

Comment:
nope, it's pretty straightforward if You read somewhere in the code `new ResponseListener` instead of `new Response` - no need to check if it's a listener, a Symfony Response class, or maybe a response compiler pass